OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -601,6 +601,21 @@ internal static void UpgradeActorRules(ModData modData, int engineVersion, ref L
 					if (node.Key == "SoundFile" && parent.Key.StartsWith("AmbientSound", StringComparison.Ordinal))
 						RenameNodeKey(node, "SoundFiles");
 
+				// Refactor Rectangle shape RotateToIsometry bool into WAngle HorizontalRotation
+				if (engineVersion < 20170505)
+				{
+					if (node.Key.StartsWith("RotateToIsometry", StringComparison.Ordinal))
+					{
+						var value = FieldLoader.GetValue<bool>("RotateToIsometry", node.Value.Value);
+						if (value == true)
+							node.Value.Value = "128";
+						else
+							node.Value.Value = "0";
```

Comment:
You may want to use a ternary here for legibility.

'node.Value.Value = value ? "128" : "0";'